packages/android_intent/android/src/main/java/io/flutter/plugins/android_intent/AndroidIntentPlugin.java

```diff
@@ -0,0 +1,85 @@
+package io.flutter.plugins.android_intent;
```

Comment:
Also: there shouldn't be an underscore in the package name. Todd is currently updating the other plugins and `flutter create --plugin` to remove that.